website/content/docs/configuration/session-recording/configure-storage-policy.mdx

```diff
@@ -0,0 +1,660 @@
+    --request GET \
+    $BOUNDARY_ADDR/v1/session-recordings?recursive=true&scope_id=global | jq
+```
+
+**Example output:**
+
+<CodeBlockConfig hideClipboard highlight="1-4,47-48">
+
+```plaintext
+{
+  "items": [
+    {
+      "id": "sr_nt5dFeBYdh",
+      "scope": {
+        "id": "global",
+        "type": "global",
+        "name": "global",
+        "description": "Global Scope"
+      },
+      "storage_bucket_id": "sb_DC8SPb9uc2",
+      "created_time": "2024-01-30T06:25:31.873628Z",
+      "updated_time": "2024-01-30T07:26:01.182513Z",
+      "start_time": "2024-01-30T06:25:32.412373Z",
+      "end_time": "2024-01-30T06:25:53.431183Z",
+      "duration": "21.018810s",
+      "type": "ssh",
+      "state": "available",
+      "endpoint": "ssh://xx.xxx.xxx.xxx:22",
+      "create_time_values": {
+        "user": {
+          "id": "u_lLWuYy70Wo",
+          "name": "admin_user",
+          "description": "Global admin user",
+          "scope": {
+            "id": "global",
+            "type": "global",
+            "name": "global",
+            "description": "Global Scope"
+          }
+        },
+        "target": {
+          "id": "tssh_cjTsFSU10p",
+          "name": "recording-target",
+          "scope": {
+            "id": "p_ocJPxzrz2p",
+            "type": "project",
+            "name": "recording-project",
+            "parent_scope_id": "o_aDkVBCDTvY"
+          },
+          "session_max_seconds": 28800,
+          "session_connection_limit": -1,
+          "egress_worker_filter": "\"recording-worker\" in \"/tags/type\"",
+          "type": "ssh",
+          "attributes": {
+            "default_port": 22
+          }
+        }
+      },
+      "authorized_actions": [
+        "reapply-storage-policy",
+        "no-op",
+        "read",
+        "download",
+        "delete"
+      ],
+      "retain_until": "2031-01-30T06:25:53.431183Z",
+      "delete_after": "2031-05-10T06:25:53.431183Z"
+    }
+  ],
+  "response_type": "complete",
+  "list_token": "GnJNt3mPNy6Cp62MUBCnZPdrQF7cvh4udEoW4CjL5k2ufvesMfKAheSJyNHmKf3zwL5aVKHT4P6q",
+  "sort_by": "created_time",
+  "sort_dir": "desc",
+  "est_item_count": 1
+}
+```
+
+</CodeBlockConfig>
+
+Verify that the `retain_until` and `delete_after` dates match the durations defined in the 'soc2-policy'.
+
+Alternatively, the recording details can also be queried directly by ID:
+
+```shell-session
+$ curl --header "Content-Type: application/json" \
+    --header "Authorization: Bearer $(boundary config get-token)" \
+    --request GET \
+    $BOUNDARY_ADDR/v1/session-recordings/sr_nt5dFeBYdh | jq
+```
+
+</Tab>
+</Tabs>
+
+<Note>
+
+  Existing recordings within a scope or its children do not automatically have new or updated polices applied to them. Policies must be re-applied to existing recordings to take effect. Refer to the [Update storage bucket policies](/boundary/docs/configuration/session-recording/update-storage-policy) page for more details.
```

Comment:
```suggestion
  Existing recordings within a scope or its children do not automatically have new or updated polices applied to them. Policies must be reapplied to existing recordings to take effect. Refer to the [Update storage bucket policies](/boundary/docs/configuration/session-recording/update-storage-policy) page for more details.
```

Reply:
I'm going to keep this as re-applied in all these docs, because I just noticed the UI has it labeled that way, as "Re-apply storage policy"

Reply:
👍 Makes sense